spec/app/models/mdm/service_spec.rb

```diff
@@ -1,5 +1,6 @@
 RSpec.describe Mdm::Service, type: :model do
   it_should_behave_like 'Metasploit::Concern.run'
+  include_context 'Rex::Text'
```

Comment:
`include_context` should be above any `it_should_behave_like` in the same context as the `it_should_behave_like` will have access to the `include_context`'s callbacks, lets, and methods.

```ruby
RSpec.describe Mdm::Service, type: :model do
  include_context 'Rex::Text'

  it_should_behave_like 'Metasploit::Concern.run'	
```